src/validation/error.messages.ts

```diff
@@ -199,6 +199,7 @@ export enum ErrorMessages {
   SELECT_DO_YOU_WANT_TO_MAKE_CHANGES_UPDATE_STATEMENT = "Select yes if all the information in this update statement is correct",
   SELECT_UPDATE_STATEMENT_VALIDATION_RESOLUTION = "Select if you want to change your statements or change the information provided in this update",
   SELECT_IF_REMOVE_SOLD_ALL_LAND_FILTER = "Select yes if the entity has disposed of all its property or land in the UK",
+  SELECT_REMOVE_NEED_TO_MAKE_CHANGES = "Select yes if you need to make changes to this overseas entity before it is removed",
```

Comment:
If the validation error message is the only thing changing here for the remove journey, I think we can probably use the `isRemove()` check to vary this.